icebox/icebox.py

```diff
@@ -5522,6 +5757,23 @@ def parse_db(text, device="1k"):
             "WEAK_PU_ENB":           (25, 27, "lutff_5/in_0"),
             "PACKAGE_PIN":           (19, 31, 1)
         }
+    },
+
+    "u4k" : {
+        ("HFOSC", (0, 21, 1)) : {
+            "CLKHFPU":                      (0, 29, "lutff_0/in_1"),
```

Comment:
Not sure whether these are supposed to be filled in or not - but the y-coordinates here are out of range and could cause import into nextpnr to fail even if you don't care about oscillators for now.

Reply:
wrong device in script...